rules/linux/lnx_network_service_scanning.yml

```diff
@@ -0,0 +1,42 @@
+action: global
+title: Linux Network Service Scanning
+id: 3e102cd9-a70d-4a7a-9508-403963092f31
+status: experimental
+description: Detects enumeration of local or remote network services.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/21
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1046/T1046.md
+falsepositives:
+  - Legitimate administration activities
+level: medium
+tags:
+  - attack.discovery
+  - attack.t1046
+---
+logsource:
+  category: process_creation
+  product: linux
+detection:
+  selection:
+    ProcessName|endswith:
+      - '/nmap'
+      - '/telnet'
+      - '/nc'
+      - '/netcat'
```

Comment:
let's separate the netcat stuff from telnet/nmap, because we still need to filter out `l` flag (commandline)

Reply:
please apply the same for macOS rule

Reply:
thank you!